recipes-multimedia/gstreamer/gstreamer1.0-plugins-nvvideo4linux2_1.14.0-r34.1.0.bb

```diff
@@ -23,6 +24,7 @@ PACKAGECONFIG ??= "libv4l2"
 PACKAGECONFIG[libv4l2] = ",,v4l-utils,tegra-libraries-multimedia-v4l"
 EXTRA_OEMAKE = "${@bb.utils.contains('PACKAGECONFIG', 'libv4l2', 'USE_LIBV4L2=1', '', d)}"
 CFLAGS += "-fcommon"
+TARGET_CC_ARCH += "${LDFLAGS}"
```

Comment:
Let's not do this.  Use [this patch](https://github.com/OE4T/meta-tegra/blob/master/recipes-multimedia/gstreamer/gstreamer1.0-plugins-nvvideo4linux2-1.14.0-r32.7.1/0000-build-fixups.patch) from master as the starting point (it will probably need some adjustment) for your makefile fixes patch, and you won't have to hack in the LDFLAGS here, or provide the `do_compile` function.

Using that patch will also make the changes needed to make libv4l2 optional.